.babelrc

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@blcham Yes we need it for tests, also it should be placed in the root folder: https://stackoverflow.com/questions/35391279/where-is-the-location-of-babelrc-file

Reply:
Sorry I was not clear here ... I understand that babel is a javascript compiler and the question is :

Why do we need it for tests and we do not need it for the application? So what compiler we are using for the library (e.g. to run storybook). I thought we decided to not use babel, although I see this:
```   
    "build:lib": "npm run build && npm run build:css && npm pack",
   "lib": "babel ./src -d ./lib"
```
What is purpose of "lib"? Only build:lib is described in readme. So before we did not need babel and now we do? I mean it is probably ok, but I just wanted to check.

Reply:
We do not use webpack anymore, but we still need babel. Actually Storybook uses babel to transpile code: https://storybook.js.org/docs/react/configure/babel#default-configuration

Also, jest can not be used without at least .babelrc:https://github.com/vuejs/vue-cli/issues/2040. However, most of it is not useful, I can clean it and keep only what is needed to run the tests


> What is purpose of "lib"? Only build:lib is described in readme. So before we did not need babel and now we do? I mean it is probably ok, but I just wanted to check.

Indeed I do not think it is useful, it can be removed